electrumsv/daemon.py

```diff
@@ -366,6 +366,10 @@ async def run_subcommand_async(self, config_options: dict[str, Any]) \
                 return f"Error: Wallet '{command_line_wallet_path}' not loaded. " \
                     "Use the 'load_wallet' daemon subcommand to load a wallet."
 
+            if wallet is not None and wallet._network is not None:
```

Comment:
What does this give us?

Reply:
`service_signup` was the main thing that was timing out in Darren's initialization bash script so this instead returns an error to the user until initial header sync is complete.

Reply:
This is not necessary and adds a limitation to create the experience that script needs, not one relevant to ESV. The approach discussed was that any user would use the TOO_EARLY response as an indication the server is ready, and create their own experience.

Reply:
Okay will remove it